sql/src/test/java/io/crate/operation/operator/OrOperatorTest.java

```diff
@@ -11,13 +11,12 @@
 
     @Test
     public void testNormalize() throws Exception {
-        assertNormalize("name or true", isLiteral(true));
-        assertNormalize("true or name", isLiteral(true));
-        assertNormalize("false or name", isField("name"));
-        assertNormalize("name or name", isFunction(OrOperator.NAME));
-
-        assertNormalize("true or 1/0", isLiteral(true));
```

Comment:
will this now cause an error? This would be another breaking change and I think one that we should not introduce.

Reply:
You're right, but `true or (1/0 = 10)` should still return `true` and not result in a failure